guides/common/modules/con_smart-proxy-networking.adoc

```diff
@@ -0,0 +1,43 @@
+[id="{smart-proxy-context}-networking"]
+= {SmartProxy} networking
+
+The communication between {ProjectServer} and hosts registered to a {SmartProxyServer} is routed through that {SmartProxyServer}.
+{SmartProxyServer} also provides {Project} services to hosts.
+
+[NOTE]
+====
+Some outgoing traffic returns to {Project} to enable internal communication and security operations.
+====
+
+ifndef::satellite[]
+In a topology with hosts connecting to a {SmartProxyServer}, 
+endif::[]
+ifdef::satellite[]
+In xref:{project-context}-topology-with-hosts-connecting-to-a-{smart-proxy-context}[], 
+endif::[]
+{SmartProxy} provides a single endpoint for all host network communications so that in remote network segments, only firewall ports to the {SmartProxy} itself must be open.
+Hosts do not need direct access to the {ProjectServer} itself.
+
+// TODO: Replace graphic with simpler graphic and reference to "Port and firewall requirements"
+ifdef::satellite[]
+[id="{project-context}-topology-with-hosts-connecting-to-a-{smart-proxy-context}"]
+.{Project} topology with hosts connecting to a {SmartProxy}
+image::common/topology-isolated-satellite.png[{ProjectName} topology with a host]
+endif::[]
+
+ifndef::satellite[]
+In a topology with hosts connecting directly to {ProjectServer}, 
+endif::[]
+ifdef::satellite[]
+In xref:{project-context}-topology-with-hosts-connecting-directly-to-{project-context}-server[], 
+endif::[]
+hosts connect to {ProjectServer} rather than a {SmartProxy}.
```

Comment:
IMHO it's both technically wrong (because hosts connect to the "integrated" Smart Proxy of Foreman Server), and not providing any insight ("If hosts do X, they do X not Y").

Reply:
I believe the point here is compare the two setups. From that perspective, to me it makes sense to say that in the first one "hosts connect to proxy" and in the second one "host don't connect to proxy". Can you see an alternative?

Reply:
Okay, I tried rewording it. Let me know what you think.

Reply:
Great. Thx.